time frame_start_time
time frame_end_time
uint32 tdc_ns_offset
uint32 tac_ns_offset
uint32 pdc_ns_offset
uint32[22] pac_ns_offset
uint32[19] electrode_ns_offset
